src/main/resources/edu/cornell/kfs/ksr/document/datadictionary/SecurityGroupMaintenanceDocument.xml

```diff
@@ -0,0 +1,82 @@
+          p:encryptDocumentDataInPersistentSessionStorage="true" p:documentTypeName="SecurityGroupDocumentType"
+          p:businessRulesClass="edu.cornell.kfs.ksr.document.validation.impl.SecurityGroupRule"
+          p:documentAuthorizerClass="org.kuali.kfs.kns.document.authorization.MaintenanceDocumentAuthorizerBase"
+          p:workflowAttributes-ref="SecurityGroupMaintenanceDocument-workflowAttributes">
+        <property name="maintainableSections">
+        	<list>
+        		<ref bean="SecurityGroupMaintenanceDocument-Edit"/>
+        		<ref bean="SecurityGroupMaintenanceDocument-Tabs"/>
+        	</list>
+        </property>
+		<property name="lockingKeys">
+			<list>
+				<value>securityGroupId</value>
+			</list>
+		</property>
+	</bean>
+	
+    <bean id="SecurityGroupMaintenanceDocument-Edit"
+          parent="SecurityGroupMaintenanceDocument-Edit-parentBean"/>
+    <bean id="SecurityGroupMaintenanceDocument-Edit-parentBean" abstract="true"
+          parent="MaintainableSectionDefinition" p:id="generalInformationSection" p:title="Security Group">
+        <property name="maintainableItems">
+            <list>
+                <bean parent="MaintainableFieldDefinition" p:name="securityGroupId" 
+                      p:defaultValueFinder-ref="securityGroupIDFinder" p:unconditionallyReadOnly="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="securityGroupName" p:required="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="securityGroupDescription" p:required="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="active" p:defaultValue="true"/>
+            </list>
+        </property>
+    </bean>
+    
+    <bean id="SecurityGroupMaintenanceDocument-Tabs" parent="SecurityGroupMaintenanceDocument-Tabs-parentBean"/>
+    <bean id="SecurityGroupMaintenanceDocument-Tabs-parentBean" abstract="true"
+          parent="MaintainableSectionDefinition" p:id="SecurityGroupTabs" p:title="Tabs">
+        <property name="maintainableItems">
+            <list>
+                <bean parent="MaintainableCollectionDefinition" p:name="securityGroupTabs"
+                      p:businessObjectClass="edu.cornell.kfs.ksr.businessobject.SecurityGroupTab">
+                    <property name="summaryFields">
+                        <list>
+                            <bean parent="MaintainableFieldDefinition" p:name="tabName"/>
+                        </list>
+                    </property>
+                    <property name="maintainableFields">
+                        <list>
+                            <bean parent="MaintainableFieldDefinition" p:name="tabId"
+                            	  p:defaultValueFinder-ref="securityGroupTabIDFinder"
+                                  p:unconditionallyReadOnly="true" p:required="true"/>
+                            <bean parent="MaintainableFieldDefinition" p:name="tabName"
+                                  p:noLookup="true"/>
```

Comment:
Does this maintenance field need to be explicitly configured as a required one?

Reply:
updated